src/stash-list/folder.vue

```diff
@@ -18,6 +18,8 @@
               :tooltip="`Open all tabs in the group and delete the group `
                       + `(hold ${bgKey} to open in background)`" />
       <Button class="remove" @action="remove" tooltip="Delete this group" />
+      <Button class="remove stashed" @action="closeGroup"
+              :tooltip="`Close all tabs in this group`" />
```

Comment:
Sounds like a good idea, the button row is getting a bit cluttered. I'm not sure which button to assign it to though, do  you mean the "Delete Group" button? Another candidate would be the "Open all tabs in this group" button, since we're essentially reversing its action, but that already has a Ctrl-Action, so it might be a bit much...

Reply:
Sorry, I should have said--yes, I was thinking of "Delete Group". As in, click to delete the group and Alt+click to close open tabs in the group.